src/ChooseArea.svelte

```diff
@@ -100,9 +99,6 @@
 <div class="left">
   <h1>Welcome to ATIP v1</h1>
   <button type="button" on:click={() => (showAbout = !showAbout)}>About</button>
-  <br />
```

Comment:
Carbon's style overrides did weird things with whitespace, forcing hacks like this previously